### PR TITLE
net/relayd: move post-install to Migrations

### PR DESCRIPTION
**Pull request by fbrendel**
Ignore the controller/view fixes. As always @fichtner was faster :smiley:

### CODE REVIEW

File: net/relayd/src/opnsense/Relayd.xml [hunk 1]
```diff
@@ -0,0 +1,326 @@
+<model>
```
Comment (fichtner) [inline net/relayd/src/opnsense/Relayd.xml]: should this be here under src/opnsense ?

File: net/relayd/Makefile [hunk 2]
```diff
@@ -3,6 +3,6 @@ PLUGIN_VERSION=		2.0.d
 PLUGIN_DEPENDS=		relayd
 PLUGIN_COMMENT=		Relayd Load Balancer
 PLUGIN_MAINTAINER=		frank.brendel@eurolog.com
-PLUGIN_REVISION=		2
+PLUGIN_REVISION=		3
```
Comment (fichtner) [inline net/relayd/Makefile]: 4 :)

File: net/relayd/src/opnsense/mvc/app/controllers/OPNsense/Relayd/Api/SettingsController.php [hunk 3]
```diff
@@ -72,8 +72,8 @@ public function getAction($nodeType = null, $uuid = null)
                 }
             }
             if ($node != null) {
-                $result[$nodeType] = $node->getNodes();
-                $result["result"] = "ok";
+                $result['relayd'] = array($nodeType => $node->getNodes());
+                $result['result'] = "ok";
```
Comment (fichtner) [inline net/relayd/src/opnsense/mvc/app/controllers/OPNsense/Relayd/Api/SettingsController.php]: here is probably the merge conflict, should be: 'ok'

File: net/relayd/src/opnsense/Relayd.xml (hunk 1 above)
Comment (fbrendel) [reply net/relayd/src/opnsense/Relayd.xml]: no, was a backup file
deleted with the third commit
ignore it please

**Comment by fichtner** [comment]
Manually merged in https://github.com/opnsense/plugins/commit/f1c6a4fc39 due to conflicts.

**Comment by fichtner** [comment]
And thanks! :)

**Pull request closed without merging**